test/functional/wallet_sendall.py

```diff
@@ -284,6 +284,12 @@ def sendall_fails_on_high_fee(self):
                 recipients=[self.remainder_target],
                 fee_rate=100000)
 
+    @cleanup
+    def sendall_fails_on_low_fee(self):
+        self.log.info("Test sendall fails if the transaction fee is too low")
+        assert_raises_rpc_error(-8, "Fee rate (0.999 sat/vB) is lower than the minimum fee rate setting (1.000 sat/vB)",
+        self.wallet.sendall, recipients=[self.recipient], options={"fee_rate": 0.999})
```

Comment:
nit: `fee_rate` is also defined as a regular argument, so no need to use `options`
```suggestion
        self.wallet.sendall, recipients=[self.recipient], fee_rate=0.999)
```

Reply:
Done